### ISSUE TITLE
General questions

### ISSUE THREAD

**Issue by dpo** [andrewrynhard/atom-latex-plus#31]
I have a few general questions about this package:

1 I'm a bit confused as to what the `latex-plus` workflow is supposed to be exactly, and it's probably my own misunderstanding. When I open a new or existing `.tex` file (even if it already has a config file associated with it), I can't access the `latex-plus` config (⌘-`'` doesn't do anything, and the option to open the config doesn't appear in the palette). Selecting `latex-plus`->`Edit` from the `Packages` menu doesn't open the config in a new tab. It's only after I compile a first time that I get access to the config.
2. While working on #30, I tried compiling a large project with an extensive bibliography. Is there some form of parallel processing going on when `latexmk` is launched? The reason I ask is that the first time around, `latexmk` fails with errors related to undefined references. Compiling a second time succeeds, which makes me think that aux files are created at the wrong point in time. Running the same `latexmk` command from the terminal always succeeds.
3. What do the `order` values do exactly (e.g., https://github.com/andrewrynhard/atom-latex-plus/blob/master/lib/config.js#L9)? While trying to add new options to the config, I obtain confusing results. You used 1, 3, 5 and 7. Are there constraints on those values?

Thanks in advance!

**Comment by andrewrynhard** [comment]
@dpo To answer your first bullet point, `latex-plus` is not activated until the `compile` command is invoked. Your solution in #30 using the `activationHooks` looks like it will fix this issue. As for your second bullet point, this should not be happening and is worth looking into. `latex-plus` should behave the same way is if you were to invoke it via terminal. If you could open another issue so we can investigate deeper that would be great! To answer your final question, the `order` is simply the order that the options are displayed in Atom.

**Comment by andrewrynhard** [comment]
@dpo closing this.
